### ISSUE TITLE
bug: Using *ngIf on item-left, item-right places it in ion-item-content

### ISSUE THREAD

**Issue by mhartington** [driftyco/ionic#5060]
_From @brandyscarney on November 18, 2015 0:8_

If you use a *ngIf on a thumbnail in an item it will place it inside of the `ion-item-content` instead of of outside of it.

![screen shot 2015-11-17 at 7 07 57 pm](https://cloud.githubusercontent.com/assets/6577830/11228954/a4a21fbc-8d5e-11e5-93ed-0440c4666758.png)

_Copied from original issue: driftyco/ionic2#574_

**Comment by mhartington** [comment]
_From @brandyscarney on November 18, 2015 19:43_

Opened an issue with angular for this after discussing with Igor: https://github.com/angular/angular/issues/5363

**Comment by mhartington** [comment]
_From @brandyscarney on November 25, 2015 20:27_

Changing the priority on this since there is a temporary workaround:

```
<template [ngIf]="true" item-left>
  <ion-avatar item-left>
    <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAAAAACH5BAAAAAAALAAAAAABAAEAAAICTAEAOw==">
  </ion-avatar>
</template>
```

cc @advany

**Comment by brandyscarney** [comment]
Removing this from a milestone, this is dependent on https://github.com/angular/angular/issues/6303

**Comment by brandyscarney** [comment]
This will be fixed when we upgrade Angular to 2.0.0-beta.13. See Breaking Changes:
https://github.com/angular/angular/blob/master/CHANGELOG.md#200-beta13-2016-03-31

**Comment by brandyscarney** [comment]
Fixed by https://github.com/driftyco/ionic/issues/6026 wooo! :grin:
